app/code/Magento/Sales/Helper/Invoice/Logo.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Magento\Sales\Helper\Invoice;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\UrlInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class Logo extends AbstractHelper
```

Comment:
Please consider moving this functionality to a separate class (aka service) with a self-explanatory name instead of using the helper approach.